### PR TITLE
Add two main function, md5 encrypt and simple sha1 encrypt

### PR DESCRIPTION
**Pull request by iamzhangxin**
## New behavior
Add two main function, md5 encrypt and sha1 encrypt，the sha1 encrypt support choose the content will be encrypt should be sorted in the dictionary order or not, and two function all provide compare with the signature provided function

### CODE REVIEW

**Comment by neroux** [comment]
I am bit confused to be honest. `MD5Util` seems to be a wrapper class that calls `SignUtil` with MD5 and also provides an equally named method to verify a hash. `SHA1Util` on the other hand takes a string array instead of a string and also does some sorting. Is that really consistent?

Also, I am not sure whether I limited such helper classes to MD5 and SHA1 in this day and age :)

File: jodd-core/src/main/java/jodd/util/SHA1Util.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+     * SHA1 encrypts function, which determines whether the incoming parameters are sorted in the dictionary order
+     * according to the incoming values, and then encrypted using the SHA1 encryption algorithm.
+     *
+     * @param str the content will be encrypt
+     * @param flag the content need sort by dictionary order or not
+     * @return the encrypt result
+     */
+    public static String encode(String[] str, boolean flag) {
+        if (flag) {
+            // 字典序排序
```
Comment (zqq90) [inline jodd-core/src/main/java/jodd/util/SHA1Util.java]: 注释不要用中文
// no chinese please

File: jodd-core/src/main/java/jodd/util/SHA1Util.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            Arrays.sort(str);
+        }
+        StringBuilder bigStr = new StringBuilder();
+        for (String message:str) {
+            bigStr.append(message);
+        }
+        // SHA1加密
```
Comment (zqq90) [inline jodd-core/src/main/java/jodd/util/SHA1Util.java]: 注释不要用中文 x2
// no chinese please

**Comment by igr** [comment]
Thank you for submission! I agree with @neroux , it make more sense to have a single class for that. And also, `FileUtil` has to be updated.

**Comment by igr** [comment]
I will refactor the code :)

**Comment by igr** [comment]
HI there! I rewrote this in a different way, and introduce a common interface for digest, hashes and encryption.

Thank you!

**Pull request closed without merging**